dbus/com.redhat.Yggdrasil1.xml

```diff
@@ -35,6 +35,39 @@
             <arg type="a{sa{ss}}" name="workers" direction="out" />
         </method>
 
+        <!--
+            MessageJournal:
+            @truncate_length: The maximum length of worker message content.
+            @message_id: Filter journal entries to only contain entries with this message id value.
+            @worker: Filter journal entries to only contain entries with this worker name.
+            @from: Filter journal entries to only contain entries from this date time value.
+            @to: Filter journal entries to only contain entries up to this date time value.
+            @persistent: Filter journal entries to only contain entries from the persistent collection of
+            journal entries spanning multiple sessions. By default, journal entries are collected from
+            the active session.
+
+            @messages: Array of dictionary objects matching the input filter parameters.
+            Each element in the array is a dictionary with key/value pairs as follows:
+            "message_id":     <string value>,
+            "sent":           <string value>,
+            "worker_name":    <string value>,
+            "response_to":    <string value>,
+            "worker_event":   <string value>,
+            "worker_message": <string value>,
+
+            Returns the set of worker messages dispatched to workers and 
+            events emitted by workers.
+        -->
+        <method name="MessageJournal">
+            <arg type="t" name="truncate_length" direction="in" />
+            <arg type="s" name="message_id" direction="in" />
+            <arg type="s" name="worker" direction="in" />
+            <arg type="s" name="from" direction="in" />
+            <arg type="s" name="to" direction="in" />
```

Comment:
Yes, 'from' and 'to' should be clarified. I like `from-time` and `to-time`. Another option is the English words "since" and "until". Those have a similar "from time" and "to time" association, respectively.

Reply:
I think I would prefer `since/until` too.. it's less jarring to read in the `--help` output.